net/zerotier/patches/0007-fix-system-include-path.patch

```diff
@@ -0,0 +1,25 @@
+From ce428394e93b8bf26fe3e87d7edd92002938bcf6 Mon Sep 17 00:00:00 2001
+From: Liangbin Lian <jjm2473@gmail.com>
+Date: Tue, 10 Jun 2025 10:53:13 +0800
+Subject: [PATCH] fix system include path
+
+`-isystem ext` makes `#include <miniupnpc/miniupnpc.h>` actually include `ext/miniupnpc/miniupnpc.h`.
+
+we should use `$(STAGING_DIR)/usr/include` as a higher priority system include path.
```

Comment:
While doing a quick look, the thing is that the issue is caused by bundling outdated version of miniupnpc in zerotier and because of that, I am in favor to close this pull request, because the solution is not exactly upstreamable and also, this is not correct fix.

Reply:
zerotier official does not support external libminiupnpc (unless it is in /usr/lib in the system), it's hack by `0001-fix-miniupnpc-natpmp-include-path.patch`. This PR is a supplement to `0001-fix-miniupnpc-natpmp-include-path.patch`.

Archlinux workaround this by simply `rm -rf ext/miniupnpc/` https://gitlab.archlinux.org/archlinux/packaging/packages/zerotier-one/-/blob/main/PKGBUILD?ref_type=heads#L31

If it's acceptable, I can `rm -f ext/miniupnpc ext/libnatpmp`.